ext/json/json.c

```diff
@@ -178,6 +178,24 @@ PHP_JSON_API zend_result php_json_decode_ex(zval *return_value, const char *str,
 }
 /* }}} */
 
+/* {{{ */
+PHP_JSON_API zend_result php_json_validate_ex(const char *str, size_t str_len, zend_long options, zend_long depth)
```

Comment:
```
ext/json/php_json.h
101:PHP_JSON_API int php_json_decode_ex(zval *return_value, const char *str, size_t str_len, zend_long options, zend_long depth);
ext/json/json.c
200:PHP_JSON_API int php_json_decode_ex(zval *return_value, const char *str, size_t str_len, zend_long options, zend_long depth) /* {{{ */
```

Also, `PHP_JSON_API bool php_json_validate_ex(const char *str, size_t str_len, zend_long options, zend_long depth);` should be declared in ext/json/php_json.h alongside, so that other modules or pecls could depend on the headers if needed and call it from C to check if json is valid.

The PHP_JSON_API macro indicates that this functionality is deliberately being made visible to other modules (e.g. can be loaded by dlsym() or equivalents)

```
#if defined(PHP_WIN32) && defined(JSON_EXPORTS)
#define PHP_JSON_API __declspec(dllexport)
#else
#define PHP_JSON_API PHPAPI
#endif
```

lgtm otherwise